plugin/input/file/worker.go

```diff
@@ -83,18 +85,23 @@ func (w *worker) work(controller inputer, jobProvider *jobProvider, readBufferSi
 					break
 				}
 
+				// \n is line separator, -1 means that file doesn't have new valid logs.
```

Comment:
what about the last line in a file?

Reply:
this works fine, we don't write empty lines.